Configurations/make-release-package.sh

```diff
@@ -115,35 +115,31 @@ if [ "$ACTION" = "" ] ; then
     # Generate zip containing the xcframework for SPM
     cd "$CONFIGURATION_BUILD_DIR/staging-spm"
     #rm -rf "$CONFIGURATION_BUILD_DIR/Sparkle.xcarchive"
-    zip -rqyX -9 "../Sparkle-for-Swift-Package-Manager.zip" *
+    ditto -c -k --zlibCompressionLevel 9 --rsrc . "../Sparkle-for-Swift-Package-Manager.zip"
```

Comment:
We have code (right above this) to extract the *.xz|bz2 archive, validate the code signature of all the binaries/bundles there. That was added due to history of archiving issues I've seen occur, particularly in GitHub Actions. Just in case, we may want to do the same thing with the SPM zip (and this partly reason I suggested changing to ditto). This may mean pulling that verification code into its own function.

Reply:
Are the SPM binaries supposed to be code signed? 

I've updated the script locally, but it throws an error:

```
/var/folders/x1/pg2m1zvx2c32tm_7pckc8fzw0000gn/T/Sparkle.c1W0L3/Build/Products/Release/staging-spm/Sparkle.xcframework: code object is not signed at all
```

Reply:
Yes they are ad-hoc signed, but I forgot that xcframework is not code bundle. Sparkle.framework is somewhere inside of it. Codesign verification on the xcframework won't make sense.

Reply:
I'll verify the individual child frameworks.